admin_ui_support/composer.json

```diff
@@ -10,6 +10,7 @@
         "phpmetrics/phpmetrics": "^2.3"
     },
     "require": {
-        "drupal/jsonapi": "^1.15"
+        "drupal/jsonapi": "^1.19",
```

Comment:
Should this be `1.22` ?

Reply:
Yes, updated(though "^" would have got us this anyways)